assemble/src/main/scripts/generate-download-script.sh

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script will generate a DEPENDENCIES listing of packaged dependencies
+
+in=target/dependencies.raw.txt
+out=target/download-dependencies
+
+cat >"$out" <<'EOF'
+#! /usr/bin/env bash
+# This script downloads the following jars, identified by their maven
+# coordinates, using the maven-dependency-plugin.
+#
+# DISCLAIMER: This is only one possible way to download a set of dependencies
```

Comment:
I think this disclaimer feeds into a part of my worry:

This feels like we're saying to users now "you should know what jars you need to run Accumulo". I understand what you're trying to warn against, but is there a better wording to it? It comes across a bit arrogant (e.g "how should users even figure out what jars they need?").

If we're making these changes and expecting that users (re-packagers) have the smarts to build their own distribution, we have to publish the information about what is required by the core of Accumulo (or at least a baseline for what dependencies are requires). There may be multiple versions of jars which satisfy Accumulo's needs (e.g. minor/bugfix versions of a commons jar), but we still know what version *did* work (that we tested against) and can advertise that.